common/notification/actions.ts

```diff
@@ -602,6 +610,20 @@ const _notificationActions = {
             appointment: sampleAppointment,
         },
     },
+    pupil_create_new_match_chat: {
+        description: 'User has clicked on new chat with a match partner',
+        sampleContext: {
+            user: sampleUser,
+            match: { id: '1' },
+        },
+    },
+    student_create_new_match_chat: {
+        description: 'User has clicked on new chat with a match partner',
```

Comment:
Tutor / Created a new chat with a match partner

The descriptions show up in the Retool UI, maybe good to distinguish them